inc/class-wp-forms-api.php

```diff
@@ -523,7 +523,10 @@ static function render_element( $element, &$values, $form = null ) {
 				$element['#content'] = null;
 				$element['#label_position'] = 'after';
 
-				if ( $element['#value'] === $element['#checked'] ) {
+				// $element['#value'] is likely a string, $element['#checked'] may be a boolean,
+				// as we are not concerned with the actual value, simply confirm that both keys'
+				// values are truthy
+				if ( $element['#value'] && $element['#checked'] ) {
```

Comment:
But '#checked' defaults to `'1'`, which is not strictly a boolean, but a truthy string. I think the logic here was that we would be able to specify any value for `#checked` and want to ensure that that exact value is what's submitted in the form or in the given values to consider it 'checked'. This change loosens it up, meaning we couldn't use non-truthy values like '0' for '#checked', since it's merely the presence of the checkbox value, regardless of what it is, that indicates that the box is checked. The uses cases for falsey values of `#checked` seem insignificant, but I don't know if this something we truly want to do.